addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -88,11 +88,34 @@ export class MessageList extends Component {
                 threadViewer: threadView && threadView.threadViewer,
             };
         });
+        /**
+         * This observer will trigger each time the root element is resized and
+         * adjust the scroll position.
+         */
+        this.resizeObserver = new ResizeObserver(elements => {
+            // In this case, this loop seems useless since we are really working
+            // on a single element, but it's required by the ResizeObserver to
+            // avoid error: ResizeObserver loop completed with undelivered
+            // notifications.
+            for (const element of elements) {
+                if (this.threadView && this.threadView.isAtBottom) {
+                    this._adjustScrollForExtraMessagesAtTheEnd();
+                }
+            }
+        });
         // useUpdate must be defined after useRenderedValues to guarantee proper
         // call order
         useUpdate({ func: () => this._update() });
     }
 
+    mounted() {
+        this.resizeObserver.observe(this.el);
```

Comment:
Isn't it supposed to be observing `_getScrollableElement()` instead? (Which to make it worse can theoretically dynamically change reference without notice.)
I think this works "by luck" because `this.el` is always `_getScrollableElement()` in the case that makes the issue.